config/electronBuilder/electronBuilder.ghactions.config.js

```diff
@@ -0,0 +1,11 @@
+const baseElectronBuilderConfig = require('./electronBuilder.local.config');
+
+module.exports = {
+    ...baseElectronBuilderConfig,
+    publish: [{
+        provider: 's3',
+        bucket: process.env.ELECTRON_ENV === 'staging' ? 'staging-expensify-cash' : 'expensify-cash',
+        channel: 'latest',
+    }],
+    afterSign: './desktop/notarize.js',
```

Comment:
Were there any problems when this was part of the single config?
I have no problems building local builds without this change?

E.g. I don't use `--publish` and this just creates a local build for me, no need for `desktop-build-local` and `desktop-build-local-staging` scripts

---

Separate question: If `notarize.js` is something only used for config/build signing why keep it in `desktop` ? it gets packed to the electron app due to the `desktop/*.js` pattern

Reply:
> Were there any problems when this was part of the single config?

I always run into issues with the notarizing step. You don't experience that?

> Separate question: If notarize.js is something only used for config/build signing why keep it in desktop ?

Just because it's specific to the desktop application.